hostapp.go

```diff
@@ -9,38 +9,44 @@ import (
 	"path/filepath"
 	"runtime"
 
-	_ "github.com/docker/docker/daemon/graphdriver/aufs"
-	_ "github.com/docker/docker/daemon/graphdriver/overlay2"
 	"github.com/docker/docker/layer"
 	"github.com/docker/docker/pkg/idtools"
 	"golang.org/x/sys/unix"
+
+	_ "github.com/docker/docker/daemon/graphdriver/aufs"
+	_ "github.com/docker/docker/daemon/graphdriver/overlay2"
 )
 
 type HostConfig struct {
 	Labels map[string]string `json:"Labels"`
 }
 
 type Config struct {
-	ID         string `json:"ID"`
-	Image      string `json:"Image"`
 	HostConfig `json:"Config"`
-	Name       string `json:"Name"`
-	Driver     string `json:"Driver"`
+
+	ID     string `json:"ID"`
+	Image  string `json:"Image"`
+	Name   string `json:"Name"`
+	Driver string `json:"Driver"`
 }
 
 type Container struct {
 	Config
```

Comment:
is this is a subset of the type [here](https://pkg.go.dev/github.com/docker/docker@v17.12.0-ce-rc1+incompatible/api/types/container?tab=doc#Config) + HostConfig

I wonder if we should call it container, as that's slightly misleading... The `label` argument of [`Mount`](https://github.com/balena-os/hostapp/blob/master/hostapp.go#L166)
also references a label on a container _image_, right?

I mean concept wise it's true so I don't mind really

Reply:
I just realized that this still needs setup before... the container referenced here still needs to be created by the engine-host daemon right?

Reply:
hi @robertgzr yes, the container it references need to be created by the engine-host daemon. Would you prefer a different name?